arduino/src/FileHandler.cpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file FilesHandler.cpp
+ * @author yarin
+ * @brief 
+ * @version 0.1
+ * @date 2021-01-04
+ * 
+ * @copyright Copyright (c) 2021
+ * 
+ */
+#include <string>
+#include <spdlog/spdlog.h>
+#include <spdlog/sinks/basic_file_sink.h>
+#include "fileHandler.h"
+#include <ctime>
+#define FILE_NAME "../dataFromArduino/sensor"
+#define FILE_NAME_END ".log"
+
+FilesHandler::FilesHandler(int amount)
+{
+    for (int i = 0; i < amount; i++)
+    {
+        addFile(makeFileName(i,FILE_NAME,FILE_NAME_END));
+    }
+}
+FilesHandler::FilesHandler()
+{
+    for (int i = 0; i < 5; i++)
+    {
+        addFile(makeFileName(i,FILE_NAME,FILE_NAME_END));
+    }
+}
```

Comment:
```suggestion
FilesHandler::FilesHandler(): FilesHandler(5) {}
```
You can do this instead or as I mention above

Reply:
f